blockly-ws-search/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "author": "Blockly Team",
```

Comment:
The custom is name, version and description first, then everything else can be ordered however.